content/tutorials/generic/micropython-installation/micropython-installation.md

```diff
@@ -0,0 +1,205 @@
+
+## Goals
+
+At the end of this guide you will have flashed a new firmware to your Arduino board, letting you connect to it with the Arduino Lab for MicroPython. 
+
+## Requirements
+
+To follow along with this article and get up and running with MicroPython on your board you are gonna require the following material:
+- [An Arduino Nano 33 BLE](https://docs.arduino.cc/hardware/nano-33-ble) / [Arduino Nano RP2040 Connect](https://docs.arduino.cc/hardware/nano-rp2040-connect) / [Arduino Portenta H7](https://docs.arduino.cc/hardware/portenta-h7)
+- [Arduino Lab for MicroPython IDE](https://docs.arduino.cc/micropython/)
+- [Firmware file](https://docs.arduino.cc/micropython/)
+
+## Arduino Nano 33 BLE
+
+The process for flashing the firmware on the Nano 33 BLE requires to first update the bootloader and SoftDevice. Once this is done you can flash the MicroPython firmware to your board.
+
+### Installing the core
+
+Start by making sure that you have the respective core installed. Open the IDE and navigate to the boards manager. Search for your board, and make sure you have the latest version of the core installed. 
+
+![Board manager](assets/boardmanager.png)
+
+This is not only needed to upload the next sketch, but you're also going to dive into the core files themselves to find a specific tool in an upcoming step.
+
+### Update bootloader
+
+In the Nano 33 BLE core is an example sketch that you will use to update the bootloader and SoftDevice of your board. Navigate to `File > Examples > Nano33BLE_System > Nano33_updateBLandSoftDevice` and open the sketch.
+
+![Example sketch](assets/examplesketch.png)
+
+Upload the sketch to your board. Beware! the bootloader is not updated just yet! Once the sketch is uploaded, you can interface with your board through the serial monitor to actually update the bootloader.
+
+In the Serial monitor, you are asked to confirm that you want to update the bootloader. In the message text box in the top of the serial monitor, write a **"y"** and press enter to send it to your board. 
+
+You can now watch as the bootloader update process progresses, **do not disconnect or reset your board during this process** or you could brick your board.
+
+Once this bar completes, you will be prompted with another choice. This time it asks if you want to update the SoftDevice on your board. Repeat the process of sending a **"y"** to the board, and watch the progress of this too.
+
+When this bar fills and the SoftDevice update completes, the board will restart, and you've successfully made it through the procedure. 
+
+>**Note:** An important thing to do at this point which is easily forgotten is to close the serial monitor, we will need to use the serial port for other things very soon and if you leave the monitor open it will be busy and won't be able to receive any new interactions.
```

Comment:
```suggestion
***Note: An important thing to do at this point which is easily forgotten is to close the serial monitor, we will need to use the serial port for other things very soon and if you leave the monitor open it will be busy and won't be able to receive any new interactions.***
```